packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -267,6 +270,44 @@ export class ProjectService {
         return explores;
     }
 
+    async getAllExploresSummary(
+        user: SessionUser,
+        projectUuid: string,
+        filtered: boolean,
+    ): Promise<SummaryExplore[]> {
+        const explores = await this.getAllExplores(user, projectUuid);
+        const allExploreSummaries = explores.map<SummaryExplore>((explore) => {
+            if (isExploreError(explore)) {
+                return {
+                    name: explore.name,
+                    tags: explore.tags,
+                    errors: explore.errors,
+                };
+            }
+            return {
+                name: explore.name,
+                tags: explore.tags,
+            };
+        });
+
+        if (filtered) {
```

Comment:
@ZeRego when are requests not filtered?

Reply:
I imagine that if I've configured my project to exclude certain explores, I can never get them back. i.e. the filtering happens at compile time, not here

Reply:
For the tables configuration page, I need all the explores for the auto complete inputs.

Reply:
👍 didn't think about that. Seems like two different operations:

1. List all models in dbt project
2. List all explores in Lightdash project

But this is simple! Let's do it